app/src/java/java/co/omise/android/example/CheckoutActivity.java

```diff
@@ -101,7 +107,7 @@ public void onRequestSucceed(@NotNull Capability model) {
 
             @Override
             public void onRequestFailed(@NotNull Throwable throwable) {
-                snackbar.setText(StringsKt.capitalize(throwable.getMessage())).show();
+                snackbar.setText(Objects.requireNonNull(capitalize(throwable.getMessage()))).show();
```

Comment:
ref: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/capitalize.html
The problem is that `throwable.getMessage()` itself can return `null`. So if we change our capitalize function to only accept non null values we will have to wrap `throwable.getMessage()` in require not null instead of wrapping the entire `capitalize` function. Either way we will have to put `requireNotNull` on one of the functions.
- Example
New code:
```java
    @NonNull
    public static String capitalize(@NotNull String input) {
        if (input.isEmpty()) {
            return input;
        }

        char firstChar = input.charAt(0);
        char capitalizedFirstChar = Character.toUpperCase(firstChar);

        return capitalizedFirstChar + input.substring(1);
    }
snackbar.setText(capitalize(Objects.requireNonNull(throwable.getMessage()))).show();
```

Reply:
Great, thanks